cypress/e2e/modules/registration/registrationsHeaders.js

```diff
@@ -0,0 +1,35 @@
+import {Registration} from '../../pages/registrationPage';
+
+let registration = new Registration();
+
+export function checkHeaders(){
+    registration.getTitle().should('have.text', 'Register')
+    registration.getName().should('have.text', 'Full name:')
+    registration.getEmail().should('have.text', 'Email address:')
+    registration.getPassword().should('have.text', 'Password:')
+    registration.getRePassword().should('have.text', 'Repeat password:')
+}
+
+// let selectors = [
+//     registration.getEmail(),
+//     registration.getPassword(),
+//     registration.getRePassword()
+// ];
+
+// let text = [
+//     ' Email is required! ',
+//     ' Password is required! ',
+//     ' Password confirmation is required! '
+// ]
+
+export function verifyFormElements(selector, errorMessage) {
+    selector.click();
+    registration.getTitle().click();
+    selector.next().shouald('have.text', errorMessage);
+}
```

Comment:
I rewrote a bit this function
```suggestion
export function verifyFormElements(selector, errorMessage) {
    // selector.click();
    cy.get(selector).click();
    registration.getTitle().click();
    cy.get(selector).next().should('have.text', errorMessage);
}

```